app/javascript/components/app.jsx

```diff
@@ -11,10 +11,28 @@ import { persistCache } from 'apollo-cache-persist';
 import WidgetController from '@components/widget-controller';
 import versionCompare from '@components/version-compare';
 import helioSchema from '@javascript/schema.json';
+import { fetch } from 'isomorphic-fetch';
+import { from } from 'apollo-link';
+import { onError } from 'apollo-link-error';
 import GlobalStyle from '../styles';
 
 let link = null;
 
+// eslint-disable-next-line new-cap
+export const errorLink = new onError(
+  ({ operation, response, graphQLErrors, networkError }) => {
+    if (graphQLErrors)
+      graphQLErrors.map(({ message, locations, path }) =>
+        // eslint-disable-next-line no-console
+        console.log(
+          `[GraphQL Error]: Message: ${message}, Location: ${locations}, Path: ${path}, Operation: ${operation}, Response: ${response}`,
+        ),
+      );
+
+    if (networkError) console.log(`[Network Error]: ${networkError}`); //eslint-disable-line
```

Comment:
Nit: I'm surprised this isn't automatically fixed by prettier. Can you use explicit `{ .. }` blocks for both of these `if` statements?

Reply:
Resolved!